src/webhooks/registry.ts

```diff
@@ -202,15 +216,22 @@ function buildQuery(
 }
 
 const WebhooksRegistry: RegistryInterface = {
-  webhookRegistry: [],
+  webhookRegistry: {},
+
+  addHandler({path, topic, webhookHandler}: WebhookRegistryEntry): void {
+    WebhooksRegistry.webhookRegistry[topic] = {path, topic, webhookHandler};
+  },
```

Comment:
Since `topic` is now the index into the hash, it's no longer needed in the `WebhookRegistryEntry` type itself, i.e., in the `webhooks/types.ts` file, the interface can be changed to
```
export interface WebhookRegistryEntry {
  path: string;
  webhookHandler: WebhookHandlerFunction;
}
```

I've had a look through the rest of the code in `registry.ts` and `<entry>.topic` isn't used anywhere, as far as I can see.

Reply:
I don't think I can remove it because in `addHandler` I need the `topic` to use as a key in the `WebhooksRegistry.webhookRegisry` object

```js
  addHandler({path, topic, webhookHandler}: WebhookRegistryEntry): void {
    WebhooksRegistry.webhookRegistry[topic] = {path, topic, webhookHandler};
  },
```

what I can do instead is remove the topic from the values since it's redundant